### ISSUE TITLE
React Native 0.70.5 Build Errors

### ISSUE THREAD

**Issue by OskarEichler** [wix/react-native-navigation#7631]
## 🐛 Bug Report

React Native: 0.70.5
React Native Navigation: 7.30.0
compileSdkVersion = 33

Building Android on the latest version we are getting the following error:

```
e: /node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/stack/topbar/button/ButtonPresenter.kt: (82, 36): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type View?
```

When fixing the error manually by adding a ? we are getting another error

```
e: /node_modules/react-native-navigation/lib/android/app/src/main/java/com/reactnativenavigation/views/animations/BaseViewAnimator.kt: (46, 9): 'onAnimationStart' overrides nothing
```

Any clues?

**Comment by OskarEichler** [comment]
Might be related to this: https://github.com/wix/react-native-navigation/pull/7558

**Comment by OskarEichler** [comment]
For now reducing compileSdkVersion to 32 resolved it.

**Comment by OskarEichler** [comment]
7.30.1 fixed it by merging #7558 🙌🏼
